tests/ResultTest.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Xenolope\Quahog\Tests;
+
+use PHPUnit\Framework\TestCase;
+use Xenolope\Quahog\Result;
+
+class ResultTest extends TestCase
+{
+    public function testCleanResult(): void
+    {
+        $result = new Result('OK', 'filename', null, null);
+
+        self::assertSame('filename', $result->getFilename());
```

Comment:
In a future PR, we can introduce an object comparation for the tests.

Reply:
Sounds good